name,value,relevant log,explanation
io.bytes.per.checksum,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
dfs.client.block.write.retries,3,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
dfs.datanode.max.nodes.to.report,5,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null
	",value-hit
io.bytes.per.checksum,0,"ERROR	Task: attempt_1701312551309_0221_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
ipc.[port_number].weighted-cost.handler,1,"ERROR	Task: attempt_1701312551309_0221_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
yarn.resourcemanager.placement-constraints.algorithm.pool-size,1,"ERROR	Task: attempt_1701312551309_0221_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
dfs.client.block.write.retries,3,"ERROR	Task: attempt_1701312551309_0221_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
dfs.datanode.max.nodes.to.report,5,"ERROR	Task: attempt_1701312551309_0221_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
io.bytes.per.checksum,0,"INFO	Diagnostics report from attempt_1701312551309_0221_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
ipc.[port_number].weighted-cost.handler,1,"INFO	Diagnostics report from attempt_1701312551309_0221_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
yarn.resourcemanager.placement-constraints.algorithm.pool-size,1,"INFO	Diagnostics report from attempt_1701312551309_0221_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
dfs.client.block.write.retries,3,"INFO	Diagnostics report from attempt_1701312551309_0221_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
dfs.datanode.max.nodes.to.report,5,"INFO	Diagnostics report from attempt_1701312551309_0221_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:0, effective=null	",value-hit
ipc.[port_number].weighted-cost.handler,1,INFO	1 failures on node ed761ad59d44,value-hit
yarn.resourcemanager.placement-constraints.algorithm.pool-size,1,INFO	1 failures on node ed761ad59d44,value-hit
dfs.datanode.max.nodes.to.report,5,INFO	1 failures on node ed761ad59d44,value-hit
io.bytes.per.checksum,0,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	io
ipc.[port_number].weighted-cost.handler,1,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	handler
dfs.datanode.max.nodes.to.report,5,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	to
io.bytes.per.checksum,0,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	io
ipc.[port_number].weighted-cost.handler,1,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	handler
dfs.datanode.max.nodes.to.report,5,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	to
io.bytes.per.checksum,0,"WARN	Graceful stop failed. Exiting..java.lang.NullPointerException
	",property-item-hit	io
dfs.datanode.max.nodes.to.report,5,"WARN	Graceful stop failed. Exiting..java.lang.NullPointerException
	",property-item-hit	to
io.bytes.per.checksum,0,INFO	Exiting with status 1: java.lang.NullPointerException,property-item-hit	io
ipc.[port_number].weighted-cost.handler,1,INFO	Exiting with status 1: java.lang.NullPointerException,value-hit
yarn.resourcemanager.placement-constraints.algorithm.pool-size,1,INFO	Exiting with status 1: java.lang.NullPointerException,value-hit
